packages/playground/src/views/settings.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <view-layout>
+    <v-card color="primary" class="d-flex justify-center items-center mt-3 pa-3 text-center">
+      <v-icon size="30" class="pr-3">mdi-cog</v-icon>
+      <v-card-title class="pa-0">Settings</v-card-title>
+    </v-card>
+    <v-card class="my-5"
+      ><v-card-title>Theme</v-card-title> <v-card-text>Pick an application theme!</v-card-text>
```

Comment:
Is this needed? Even if the user didn't choose a theme, the first value will always be chosen.

Reply:
no, the current theme is chosen.